.eslintrc.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
   root: true,
   extends: ['@metamask/eslint-config', '@metamask/eslint-config-nodejs'],
+  parserOptions: {
+    sourceType: 'module',
+  },
```

Comment:
to fix this issue
<img width="661" alt="Screenshot 2024-10-15 at 4 35 18 PM" src="https://github.com/user-attachments/assets/b82ba084-d454-41f7-954f-307c7b081b8e">